tests/utils/k8s.go

```diff
@@ -66,9 +69,64 @@ func DropNamespace(clientSet *kubernetes.Clientset, name string) error {
 	})
 }
 
+func getPodLog(clientSet *kubernetes.Clientset, namespace string, pod string, lines int64) (string, error) {
+	opts := v1.PodLogOptions{}
+	if lines > 0 {
+		opts.TailLines = &lines
+	}
+	req := clientSet.CoreV1().Pods(namespace).GetLogs(pod, &opts)
+
+	logSteam, err := req.Stream()
+	if err != nil {
+		return "", err
+	}
+	defer logSteam.Close()
+
+	logBuffer := new(bytes.Buffer)
+	_, err = io.Copy(logBuffer, logSteam)
+	if err != nil {
+		return "", err
+	}
+
+	return logBuffer.String(), nil
+}
+
+func podLogContains(clientSet *kubernetes.Clientset, namespace string, pod string, text string) (bool, error) {
+	opts := v1.PodLogOptions{}
+	req := clientSet.CoreV1().Pods(namespace).GetLogs(pod, &opts)
+
+	logSteam, err := req.Stream()
+	if err != nil {
+		return false, err
+	}
+	defer logSteam.Close()
+
+	scanner := bufio.NewScanner(logSteam)
+	for scanner.Scan() {
+		if strings.Contains(scanner.Text(), text) {
+			return true, nil
+		}
+	}
+
+	if err = scanner.Err(); err != nil {
+		return false, err
+	} else {
+		return false, nil
+	}
+}
+
+func logPodLogTail(clientSet *kubernetes.Clientset, namespace string, pod string, lines int64) error {
```

Comment:
and here, `lines` fits great as a name because the function name implies that this is the number of lines to tail